opentracing/tracer.py

```diff
@@ -34,44 +33,53 @@ class Tracer(SpanPropagator):
 
     singleton_noop_span = Span()
 
-    def start_trace(self, operation_name=None, tags=None):
-        """Starts a new trace and creates a new root span.
+    def start_span(self, operation_name, parent=None, **kwargs):
```

Comment:
the idea is to make sure that we don't lock in to an argument order... arguments about the "compiler" in python are a little odd to me -- you mean the linter?